### PR TITLE
Add an example, link it in the README

### PR DESCRIPTION
**Pull request by jmacd**
**Description:** Add an example to test the smallest configuration of the Lightstep metrics SDK for stand-alone use.

**Testing:** This is a test.

**Documentation:** Linked from README.md

### CODE REVIEW

**Review by mwear — COMMENTED** [review]
LGTM, but there is another reference to `test.NewServer` in `examples/tracing/trace_test.go` that is passing in an argument. See: https://github.com/lightstep/otel-launcher-go/pull/219/files#file-examples-tracing-trace_test-go-L27.

**Comment by jmacd** [comment]
Fixed, PTAL.

**Review by jaronoff97 — APPROVED** [review]
left one non-blocking comment, otherwise this looks great. Thanks for putting this together!

File: lightstep/sdk/metric/example/example_test.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	))
+
+	// Configure the SDK.
+	sdk := lightstep.NewMeterProvider(
+		lightstep.WithReader(lightstep.NewPeriodicReader(exp, 30*time.Second)),
+	)
+
+	// Configure a Meter and instrument.
+	meter := sdk.Meter("meter")
+	counter, _ := meter.SyncInt64().Counter("how-many")
```
[the remark this answers is not in the document]
Comment (jmacd) [reply lightstep/sdk/metric/example/example_test.go:55]: The top-level examples directory has examples of using each kind instrument in a `main` program.

**Pull request merged**